zerver/webhooks/ansibletower/doc.md

```diff
@@ -0,0 +1,19 @@
+Get Ansible Tower notifications in Zulip!
+
+ 1. {!create-stream.md!}
+
+ 1. {!create-bot-construct-url-indented.md!}
+
+ 1. Go to your Ansible Tower or AWX Admin Portal, and click **Notifications** on the left sidebar.
+    Click **Add**.
+
+ 1. Set **Name** to a name of your choice, such as `Zulip`. Select the organization
+    you'd like to be notified about, and set **Type** to **Webhook**. Set **Target URL**
+    to the URL constructed above, and click **Save**.
+
+ 1. Click on **Organizations** on the left sidebar. Click the pencil icon to edit the organization you selected
+    above, and click **Notifications**. Select the events you would like to be notified about, and click **Save**.
+
+ {!congrats.md!}
+
+ ![](/static/images/integrations/ansibletower/001.png)
```

Comment:
I'm being a bit nitpicky here, but could you please wrap the text at 80 characters? I've tried my best to make sure we wrap the text in all the docs. :)